app/controllers/organization_account_requests_controller.rb

```diff
@@ -0,0 +1,34 @@
+class OrganizationAccountRequestsController < ApplicationController
+  skip_before_action :authenticate_user!
+  skip_verify_authorized
+
+  def new
+    @organization_acccount_request = Organization.new
+  end
+
+  def create
+    @organization_acccount_request = Organization.new(organization_params)
```

Comment:
```suggestion
  @contact = Contact.new(contacts_params)
```

Reply:
One thing we might want to think about is our current Contact model validates for name, email, message but in this case we will want to validate for different fields.

Reply:
Yes we will need a new model! I mention in my comment below.

Reply:
My train of thought was to use the Contact model as we already use it for `contacts_controller` and `dev_contacts_controller` (being renamed to feedback in #1051  ) and conditionally handle the validations in one model. A new model works a well. 

We will be adding a subject field in #1058 so that could use the same Contact model or would need a new model if we are separating them all out by controller. That will put us at 3 models. (Contact, feedback, OrganizationAccountRequest)

Reply:
Yes we could use one. I feel like it would get outgrown quickly though and more of a fan of a dedicated model per form (we can reuse the same model for like forms). We could also move them to a specific folder under app/models to keep them clear and separate from other models that have tables associated.

Reply:
Thanks everyone for the feedback, I added a new `OrganizationAccountRequest` model.